### PR TITLE
[DC-1661] Exclude death table from cleaning rule to drop zero concept ids

### PR DESCRIPTION
**Pull request by nishanthpp93**
* [DC-1661] Exclude death table

[DC-1661]: https://precisionmedicineinitiative.atlassian.net/browse/DC-1661

### CODE REVIEW

File: data_steward/cdr_cleaner/cleaning_rules/drop_zero_concept_ids.py [hunk 1]
```diff
@@ -26,12 +26,13 @@
...
-death
-    - cause_source_concept_id
-    - cause_concept_id
 
 Remove those rows from the clean dataset
 Archive/sandbox those rows
+
+As of DC-1661, the death table has been removed.
+This allows the death table with suppressed cause_concept_id and
+cause_source_concept_id to persist without being deleted.9
```
Comment (dcampbell-vumc) [inline data_steward/cdr_cleaner/cleaning_rules/drop_zero_concept_ids.py:35]: Small typo at end of sentence

```suggestion
cause_source_concept_id to persist without being deleted.
```

**Review by dcampbell-vumc — CHANGES_REQUESTED** [review]
Small typo in the doc string comment. The unit test runs as expected however I think we should update the unit test doc string and remove the reference to the death table as well as add the comment you put in the cleaning rule doc string about the change reasoning. That could prevent any future confusion.

File: data_steward/cdr_cleaner/cleaning_rules/drop_zero_concept_ids.py (hunk 1 above)
Comment (nishanthpp93) [reply data_steward/cdr_cleaner/cleaning_rules/drop_zero_concept_ids.py]: Thanks for catching this, I updated it

**Review by dcampbell-vumc — APPROVED** [review]
Thanks for making those changes! Unit test ran successfully so this is good to go!

**Pull request merged**